### ISSUE TITLE
Is it possible that this library gets way to set Activity Types?

### ISSUE THREAD

**Issue by WildlandHerald** [discordjs/RPC#146]
As per [discord docs](https://discord.com/developers/docs/topics/gateway#activity-object-activity-types) it is possible to set Activity Type somehow, for example - i want to use Listening (enum 2) type in my application. Is it possible to implement this in library?

**Comment by Terroriser1** [comment]
Yeah, I am also interested in this feature for the Watching type, good suggestion.

**Comment by ItsSunnyMonster** [comment]
Is there any reason why this can't be implemented?

**Comment by JoCat** [comment]
> Is there any reason why this can't be implemented?

Yes, discord ignores any passed value of type.

_I think you've already figured it out, then I'll just leave this post for those who, like me, have encountered this problem so they don't have to search for information._

**Comment by Araxeus** [comment]
Activity Types are now supported in discord rpc - see https://x.com/advaithj1/status/1816655583107510517  (Jul 26, 2024)

![image](https://github.com/user-attachments/assets/42ca1013-a762-4056-99d6-3ae3856a8d74)

code example: https://github.com/th-ch/youtube-music/blob/f2b1e6b6bfa748c82c144d76140248c77bea7569/src/plugins/discord/main.ts#L183-L190
 
[ActivityType](https://discord.com/developers/docs/topics/gateway-events#activity-object-activity-types) typescript definition here: https://github.com/discordjs/discord-api-types/blob/a59f9f002f2fad7cdc6ce8aa9cf04081db2e0da8/payloads/v10/gateway.ts#L251-L276

**Comment by JoCat** [comment]
Yes, I agree, it seems to be working now.
![image](https://github.com/user-attachments/assets/692383f0-c01b-416a-b399-634245b923dd)
